hp_list/main.c

```diff
@@ -364,6 +475,10 @@ static void *delete_thread(void *arg)
 
 int main(void)
 {
+#ifdef RUNTIME_STAT
```

Comment:
Use another macro to wrap, such `RUNTIME_STAT_INIT()`. So that, you can eliminate the `#ifdef` ... `#endif` here.